process/transaction/baseProcess.go

```diff
@@ -137,7 +137,7 @@ func (txProc *baseTxProcessor) checkTxValues(
 		return process.ErrWrongTypeAssertion
 	}
 
-	txFee := txProc.economicsFee.ComputeFee(tx)
+	txFee := core.SafeMul(tx.GasLimit, tx.GasPrice)
```

Comment:
So, now for this case the txFee will be calculated by directly multiply gasPrice with gasLimit which has been put inside tx and not by applying the below formula for gas limit, as was done before?

```
	gasLimit := ed.minGasLimit
	dataLen := uint64(len(tx.GetData()))
	gasLimit += dataLen * ed.gasPerDataByte

```

Reply:
In this case line 150 could be replaced with cost.Add(cost, txFee) to avoid a multiply operation on big.Int which already has been done?

Reply:
done